data.sql

```diff
@@ -0,0 +1,30 @@
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_in_kg) 
+VALUES 
+(
+  'Agumon',
+  '2020-02-03',
+  0,
+  '0',
+  10.23
+),
```

Comment:
- [ ] Kindly confirm all data entries are correct, according to the requirements this animal's neutered value should be true as indicated on the bold text below.
`His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He **was neutered** and he has never tried to escape.